pkgs/servers/web-apps/goatcounter/default.nix

```diff
@@ -13,20 +13,22 @@
 
 buildGoModule rec {
   pname = "goatcounter";
-  version = "2.4.1";
+  version = "2.5.0";
 
   src = fetchFromGitHub {
     owner = "arp242";
     repo = "goatcounter";
     rev = "v${version}";
-    hash = "sha256-L4310L+L2Qn8NkRMqze7KNwZ18LXz8PAoXCCpYa5J4I=";
+    hash = "sha256-lwiLk/YYxX4QwSDjpU/mAikumGXYMzleRzmPjZGruZU=";
   };
-  vendorHash = "sha256-nKfqZ5hGGVLBY/hnJJPCrS/9MlGoR2MWFUWDnpwWgyM=";
+  vendorHash = "sha256-YAb3uBWQc6hWzF1Z5cAg8RzJQSJV+6dkppfczKS832s=";
 
   subPackages = "cmd/goatcounter";
 
   buildInputs = [ postgresql ] ++ lib.optional enableSqlite sqlite;
 
+  doCheck = false;
```

Comment:
I'd prefer to keep the checks - they sometimes expose subtle problems that only crop up with Nix.

I grepped through `nixpkgs` and there were 36562 occurrences of `buildGoModule` - of those files that matched, 3755 had an explicit `doCheck = false`. That'd be a lower bound though since it doesn't include packages that conditionally enable/disable checks.

On my base M1 MBP the checks double the build time from about 30s to 60s. On balance, I think it's worth keeping them enabled.